docs/user/azure/README.md

```diff
@@ -0,0 +1,19 @@
+# Azure Account Set Up
+
+This document is a guide for preparing a new Axure account for use with OpenShift. It
```

Comment:
Nit: personally I prefer one sentence per line so I don't have to think about reflowing these or remember to set `--word-diff=color` when looking at diffs.  But I can live with this if you want to wrap differently ;).

Reply:
done https://github.com/openshift/installer/compare/fc6532c8637a2c37f4c0a028da1a5a430ac19302...3bb0cfefaf513837f94ff94c1bc59e047421323b

Reply:
One line per paragraph works too ;).